src/pymor/analyticalproblems/expressions.py

```diff
@@ -192,6 +222,10 @@ def __init__(self, value):
         self.numpy_symbol = repr(value.tolist())
         self.shape = value.shape
 
+    def fenics_expr(self, params):
+        from dolfin import Constant
+        return np.array(Constant(self.value))
```

Comment:
Upon tinkering around with this for a bit I noticed that the above code does indeed not work properly. In the current state it seems to concatenate the 'lists' instead of performing the expected element-wise addition (imagine UFL expressions instead of floats):
```python
>>> a = [1., 2.]
>>> b = [1., 2.]
>>> a + b
[1.0, 2.0, 1.0, 2.0]       # This is what actually happens
>>> np.array(a) + np.array(b)
array([2., 4.])            # This is the expected behaviour
```
I am going to investigate this further, but if there's an easy fix to this, please let me know.

Reply:
See #1569. I will try to take care of this soon.